src/ast/passes/codegen_llvm.cpp

```diff
@@ -184,7 +184,7 @@ void CodegenLLVM::kstack_ustack(const std::string &ident,
 
   std::vector<llvm::Type *> stack_key_elements = {
     b_.getInt64Ty(), // stack id
-    b_.getInt32Ty(), // nr_stack_frames
+    b_.getInt64Ty(), // nr_stack_frames
```

Comment:
I was having some issues with the stack id when setting the map key type as a tuple with a int64 and a int32 - maybe an alignment thing?

Reply:
Yeah, it's because of padding. I could use an array here instead of a padded struct for the key if we wanted to save 4 bytes, which maybe is worth it for when we have a lot of stacks.